tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Query/QueryProjectPropertiesContextTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using Xunit;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Query
+{
+    public class QueryProjectPropertiesContextTests
+    {
+        [Theory]
+        [MemberData(nameof(ContextsThatAreEqual))]
+        public void Equal(IProjectPropertiesContext a, IProjectPropertiesContext b)
+        {
+            Assert.True(a.Equals(b));
```

Comment:
`Equals` overrides are supposed to be symmetric (i.e. `A==B implies B==A`) but there's no guarantee of this. I like to validate this both ways. I'm not sure if MSTest's `Assert.AreEqual` does this for you.

```suggestion
            Assert.True(a.Equals(b));
            Assert.True(b.Equals(a));
```

Ditto below in `NotEqual`.